src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -13,9 +13,7 @@ module sonic-feature{
 
     typedef feature-state {
         description "configuration to set the feature running state";
-        type string {
-            pattern "enabled|disabled|always_enabled|always_disabled";
-        }
+        type string;
```

Comment:
>string [](http://example.com/codeflow?start=13&length=6)

The design of "The feature state can be a jinja template" is weird. What is the motivation? Is it in the design doc?

Reply:
@qiluo-msft I haven't found it. It is introduced in https://github.com/Azure/sonic-buildimage/pull/6700. @tahmed-dev Could you please comment?

Reply:
@qiluo-msft Issue - https://github.com/Azure/sonic-buildimage/issues/9598